flexible/cloudsql/pom.xml

```diff
@@ -46,23 +59,51 @@
       <scope>provided</scope>
     </dependency>
     <!-- [START dependencies] -->
+    <dependency>                        <!-- http://dev.mysql.com/doc/connector-j/en/ -->
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
+      <version>6.0.5</version>
+    </dependency>
     <dependency>
       <groupId>com.google.cloud.sql</groupId>
-      <artifactId>mysql-socket-factory</artifactId>
+      <artifactId>mysql-socket-factory-connector-j-6</artifactId>
       <version>1.0.2</version>
     </dependency>
     <!-- [END dependencies] -->
   </dependencies>
   <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
     <!-- for hot reload of the web application -->
     <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
     <plugins>
+
+<!--
```

Comment:
Is this meant to be commented out? Add a comment saying when to uncomment it if it's supposed to be here.